postreise/plot/tests/test_plot_capacity_map.py

```diff
@@ -0,0 +1,58 @@
+import bokeh.plotting as plt
+import pytest
+from powersimdata.tests.mock_scenario import MockScenario
+
+from postreise.plot.plot_capacity_map import map_plant_capacity
+
+mock_plant = {
+    "plant_id": ["A", "B", "C", "D", "E", "F", "G", "H"],
+    "bus_id": [1, 2, 3, 4, 5, 6, 7, 9],
+    "lat": [29.7604, 30.2672, 26.183, 31.7619, 32.7555, 30.6280, 31.5493, 29.3013],
+    "lon": [
+        -95.3698,
+        -97.7431,
+        -98.123,
+        -106.4850,
+        -97.3308,
+        -96.3344,
+        -97.1467,
+        94.7977,
+    ],
+    "Pmax": [1000, 750, 1500, 300, 500, 300, 200, 800],
+    "type": ["coal", "ng", "coal", "coal", "wind", "ng", "coal", "solar"],
+}
+
+scenario = MockScenario({"plant": mock_plant})
+
+
+def test_map_plant_capacity():
+    canvas = map_plant_capacity(scenario, ["ng"])
+    assert isinstance(canvas, plt.Figure) is True
```

Comment:
Is it sufficient to do `assert isinstance(canvas, plt.Figure)`?

Reply:
I agree that it is not sufficient. I guess what I am to test is that we can run the function so when `powersimdata` is updated with the generalization of the generator type we see where it breaks.

Reply:
@rouille I mean we can do `assert isinstance(canvas, plt.Figure)` instead of `assert isinstance(canvas, plt.Figure) is True`.

Reply:
I see. I understood that we should test that glyphs/markers were actually added to the canvas. This kind of stuffs.

Reply:
Done